src/main/java/org/myrobotlab/sensor/TimeEncoder.java

```diff
@@ -455,4 +455,10 @@ public void removeListener(String localTopic, String otherService) {
     // TODO Auto-generated method stub
 
   }
+
```

Comment:
Same, Why override this here?  shouldn't we be using the impl from the base Service class?

Reply:
if it was a Service